# context's new WithCancelCause

Excerpt from official release notes:

> The new WithCancelCause function provides a way to cancel a context with a given error. That error can be retrieved by calling the new Cause function.
